react-pool/src/utils/globalParameters.js

```diff
@@ -99,7 +99,7 @@ const LARGE_WIDGET_TRADING_LINK = window.reactInit.largeWidgetLink;
 // Global variables that are going to be used by the website
 
 // Set true or false if you wanna use the test-net (demo enviroment) or main-net enviroment of the pool node
-export const useTestNet = false;
+export const useTestNet = true;
```

Comment:
better having testnet as true... I even had a weird error on running locally, because of that

Reply:
But with this will pool operators have to edit something to get it working on their pools? If yes, better to revert this one.

Reply:
this is outdated...sorry